store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10390,4 +10390,7 @@ TODO: delete them permanently from here
   <desc>Whether retention policy feature is enabled</desc>
 </attr>
 
+<attr id="4102" name="zimbraAliasListToHide" type="email" max="256" immutable="1" cardinality="multi" optionalIn="mailRecipient">
```

Comment:
since is missing, please add and run generate getters

Reply:
done